onnx/defs/loss/defs.cc

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) ONNX Project Contributors.
+// Licensed under the MIT license.
+
+#include "onnx/defs/function.h"
+#include "onnx/defs/schema.h"
+namespace ONNX_NAMESPACE {
+const char* reduction_doc =
+    "Type of reduction to apply to loss: none, sum, mean(default). "
+    "'none': no reduction will be applied, "
+    "'sum': the output will be summed. "
+    "'mean': the sum of the output will be divided by the number of "
+    "elements in the output.";
+
+static const char* SoftmaxCrossEntropyLoss_ver12_doc =
+    R"DOC(Loss function that measures the softmax cross entropy
+between 'scores' and 'labels'.
+The loss can be described as:
+    L = (l_1, l_2, ..., l_N), where N is the batch_size
+
+shape(scores): (N, C) where C is the number of classes, or (N, C, d1, d2,..., dk),
+        with K >= 1 in case of K-dimensional loss.
+shape(labels): (N) where each value is 0 <= labels[i] <= C-1, or (N, d1, d2,..., dk),
+        with K >= 1 in case of K-dimensional loss.
+
+The loss for one sample, l_i, can caculated as follows:
+    l_i = -y[i][c][d1][d2]..[dk], where i is the index of classes.
```

Comment:
is the LHS supposed to be l[i][d1][d2]...[dk] ?

Reply:
yes, l_i is of shape l[i][d1][d2]...[dk]

Reply:
I think it is better to change the LHS to l[i][d1]...[dk]. Otherwise, it is unclear where the d1 to dk in the RHS come from.